### ISSUE TITLE
sql password misinterpreted

### ISSUE THREAD

**Issue by Geniks** [deStrO/eBot-CSGO-Web#88]
Hello

I had as password for sql user this string : `1+TS0Mm+ZwbpfEww`
and I was able to connect from shell / phpmyadmin / etc

but when I wanted to do
`php symfony doctrine:build --all --no-confirmation`
there was an error :
`PDO Connection Error: SQLSTATE[HY000] [1045] Access denied for user 'ebotv3'@'localhost' (using password: YES)`

I tried adding "" around the password, but I didn't work so I changed it and it works
So I guess somewhere the password is misinterpreted ?

**Comment by MikkelDK** [comment]
Do you have created user "ebotv3" ?

**Comment by Geniks** [comment]
Yeah as I said I was able to connect with this user via mysql command in shell and phpmyadmin ;)
